### PR TITLE
Expand game board size to 11x11; Add letters in initial tiles

### PR DESCRIPTION
**Pull request by henghaopu**

### CODE REVIEW

File: Frontend/models/game-board-model.js [hunk 1]
```diff
@@ -1,7 +1,41 @@
...
     selectedTileParentId: '',
-    selectedSquareId: ''
+    selectedSquareId: '',
+    // can change to computed property
+    rows: generateTableRows(),
+    squares: generateSquares()
+}
+
+function generateTableRows () {
+    var row = 11;
```
Comment (BrendanOswego) [inline Frontend/models/game-board-model.js]: Please abstract these values into some config file like `constants.js` and do something such as `export const rows = 11` and `export const cols = 11` and then import the config file wherever you need those values

**Comment by landonp1203** [comment]
@BrendanOswego it’s most likely my fault Travis is failing, I was doing a quick fix and miss d punting the file. There seems to be a formatting error which is making all the builds fail/error. I will fix it right away.

**Comment by landonp1203** [comment]
@BrendanOswego How do these changes look? You're the better expert here.

**Comment by henghaopu** [comment]
Hi @BrendanOswego ,

In JavaScript, I don’t think it’s a serious issue that needed to be revised at this stage because unlike Java, JavaScript is weekly typed and the var is function scope which is an simple and early introduced feature. 

If you trace the source code of Vue.js written by the Vue inventor, you’ll find that he use var instead of const to declare a constant. I understand it’s good to use let and const which are block scope, but this should be a low priority refactoring issue comparing to a logic error. 

This is only my shallow opinion. Please feel free to correct me if I got anything wrong.

**Pull request merged**